docs/source/pagination/introduction.mdx

```diff
@@ -81,14 +81,14 @@ pager.sink { result in
 The `GraphQLQueryPager` class provides several methods to fetch paginated data: `fetch`, `refetch`, `loadNext`, `loadPrevious`, and `loadAll`.
 
 - `fetch`: Fetches the first page of data. Must be called before `loadNext` or `loadPrevious` can be called.
-- `refetch`: Cancels all ongoing fetch operations and resets the pager to its initial state. Fetches the first page of data.
+- `refetch`: Cancels all in-flight fetch operations and resets the pager to its initial state. Fetches the first page of data.
 - `loadNext`: Fetches the next page of data. Can only be called after `fetch` has been called.
 - `loadPrevious`: Fetches the previous page of data. Can only be called after `fetch` has been called.
 - `loadAll`: Fetches all pages of data. If no initial page is detected, it will first call `fetch` to fetch the first page of data. Will continue to fetch all pages until a `PageInfo` object indicates that there are no more pages to fetch. This function is compatible with forward, reverse, and bidirectional pagination.
 
 ## Cancelling ongoing requests
 
-The `GraphQLQueryPager` class provides a `cancel` method, which can be used to cancel any ongoing fetch operations as well as cease all subscriptions. Once cancelled, the pager's state is reset. Any subscriber that was subscribed to the pager will receive a termination signal, and will no longer receive updates. In order to resume fetching data, a new `GraphQLQueryPager` instance must be created.
+The `GraphQLQueryPager` class provides a `reset` method, which can be used to cancel all in-flight fetch operations and stop watching for changes. Once the pager's state is reset, you can call `fetch` to being receiving updates again.
```

Comment:
I wonder if the `stop watching for changes` is a little ambiguous? It's true – in that we stop the underlying watchers. Should we clarify that any object subscribed to the pager would remains subscribed?

Reply:
Fixed